app/lib/pre_assembly/bundle.rb

```diff
@@ -183,11 +184,12 @@ def discover_items_via_crawl(root)
     end
 
     # Returns a list of the ObjectFiles for a digital object.
-    def discover_object_files(stageable_items)
+    def discover_object_files(stageable_items, druid)
```

Comment:
This seems like a good time to extract this method into a new class like `ObjectFileFinder` or something.  I think we're hitting the point where this class is doing too much.

Reply:
@jcoyne I'm down for the refactor, but let's get this in first as it fixes a production problem;  I'll do the refactor as a separate PR.

Reply:
Justin Coyne agreed in private slack conv that refactoring can follow in separate PR.